smoke-test.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# fail this script immediately if any command fails with a non-zero exit code
+set -e
+# Treat unset env variables as an error
+set -u
+# fail on pipeline errors, e.g. when grepping
+set -o pipefail
+
+readonly ES_VERSION=7.3.0
```

Comment:
Does it make sense to allow honoring an env var already set via something like `ES_VERSION=${ES_VERSION:-7.3.0}`

Reply:
Good point. I'll push that in a follow-up commit.

Reply:
Added in https://github.com/elastic/rally-eventdata-track/pull/47/commits/9a35d01844bee51e8290232da110c5e96f8fcf40